guides/common/modules/con_troubleshooting-dnf-modules.adoc

```diff
@@ -0,0 +1,8 @@
+[id="Troubleshooting_DNF_Modules_{context}"]
+= Troubleshooting DNF Modules
+
+ifndef::foreman-deb[]
```

Comment:
The whole file should only be included on non-deb, not only those sections.

Reply:
Addressed :+1: